From 7ea31898f8f55776a78f2e1f99d120b5407fab21 Mon Sep 17 00:00:00 2001
From: Lin Huang <hl@rock-chips.com>
Date: Wed, 9 Aug 2017 08:11:19 +0800
Subject: [PATCH] CHROMIUM: arm64: dts: rockchip: set HEADSET_INT to pull up on
 Scarlet

Since on scarlet HEADSET_INT_ODL do not have external pull up, we
need to set this gpio status to internal pull up.

BUG=b:63790585
TEST=headphone can work on scarlet

Change-Id: I78a89068aa5c5d7d46761b9aa602f37b1acf2665
Signed-off-by: Lin Huang <hl@rock-chips.com>
Reviewed-on: https://chromium-review.googlesource.com/606848
Reviewed-by: Alexandru M Stan <amstan@chromium.org>
Reviewed-by: Brian Norris <briannorris@chromium.org>
---
 arch/arm64/boot/dts/rockchip/rk3399-gru-scarlet.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-gru-scarlet.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-gru-scarlet.dtsi
index f1c26e3451be..a803bc2876f7 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-gru-scarlet.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-gru-scarlet.dtsi
@@ -337,6 +337,10 @@ pp900_emmcpll: &pp900_s3 {
 	rockchip,pins = <1 17 RK_FUNC_GPIO &pcfg_pull_up>;
 };
 
+&headset_int_l {
+	rockchip,pins = <1 23 RK_FUNC_GPIO &pcfg_pull_up>;
+};
+
 /* there is no external pull up, so need to set this pin pull up */
 &sdmmc_cd_gpio {
 	rockchip,pins = <1 11 RK_FUNC_GPIO &pcfg_pull_up>;
-- 
2.20.1

